aspnetcore/blazor/javascript-interop.md

```diff
@@ -24,6 +24,7 @@ For server-side apps:
 
 * Multiple user requests are processed by the server-side app. Don't call `JSRuntime.Current` in a component to invoke JavaScript functions.
 * Inject the `IJSRuntime` abstraction and use the injected object to issue JavaScript interop calls.
+* While a Blazor server-side app is prerendering, calling into JavaScript isn't possible because a connection with the browser hasn't been established. For more information, see the [Detect when a Blazor server-side app is prerendering](#detect-when-a-blazor-server-side-app-is-prerendering) section.
```

Comment:
```suggestion
* While a Blazor app is prerendering, calling into JavaScript isn't possible because a connection with the browser hasn't been established. For more information, see the [Detect when a Blazor app is prerendering](#detect-when-a-blazor-server-side-app-is-prerendering) section.
```